modules/machineset-yaml-aws.adoc

```diff
@@ -144,8 +144,13 @@ $ oc -n openshift-machine-api \
 ====
 Custom tags can also be specified during installation in the `install-config.yml` file. If the `install-config.yml` file and the machine set include a tag with the same `name` data, the value for the tag from the machine set takes priority over the value for the tag in the `install-config.yml` file.
 ====
+
+<6> Specify the zone, for example, `us-east-1a`.
+<7> Specify the region, for example, `us-east-1a`.
```

Comment:
<7> the region example should be: us-east-1 (not us-east-1a)

Reply:
ack, updated the doc.